spec/operators/windowCount-spec.js

```diff
@@ -3,31 +3,103 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.windowCount', function () {
-  function mergeMapfunction(x) {
-    return x.toArray();
-  }
+  it('should emit windows with count 2 and skip 1', function () {
+    var source = hot('^-a--b--c--d--|');
+    var subs =       '^             !';
+    var expected =   'u-v--x--y--z--|';
+    var u = cold(    '--a--(b|)      ');
+    var v = cold(      '---b--(c|)   ');
+    var x = cold(         '---c--(d|)');
+    var y = cold(            '---d--|');
+    var z = cold(               '---|');
```

Comment:
@kwonoj @blesh Notice `z` is the empty window at the end. I noticed this is actually necessary. By the time `z` is opened, the observer simply does not know what will happen next, whether it'll be `next` or `complete`. And notice window `u` starts immediately at the beginning. This is to stay consistent with `windowTime` and `window` operators that do the same.